cmd/gotpm/main.go

```diff
@@ -2,12 +2,42 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"runtime/debug"
 
 	"github.com/google/go-tpm-tools/cmd"
 )
 
+// GoReleaser will populates those fields
```

Comment:
how does it do this? I see https://pkg.go.dev/runtime/debug#ReadBuildInfo, does GoReleaser fill that out?

Reply:
yes, by default these three will be populated https://goreleaser.com/cookbooks/using-main.version/